_docs_agent/plugins/dropwizard.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: page
+title: Dropwizard plugin
+description: Information on the metrics collected by the CoScale Dropwizard plugin.
+---
+
+> Dropwizard is a Java framework for developing ops-friendly, high-performance, RESTful web services. Dropwizard pulls together stable, mature libraries from the Java ecosystem into a simple, light-weight package that lets you focus on getting things done. Dropwizard has out-of-the-box support for sophisticated configuration, application metrics, logging, operational tools, and much more, allowing you and your team to ship a production-quality web service in the shortest time possible.
+
+More information on: [http://www.dropwizard.io](http://www.dropwizard.io)
+
+## How it works
+
+This plugin analyzes the performance of your Dropwizard web application. It captures traffic rates, latencies and response codes of http requests sent to the web server by parsing the Dropwizard access log file. The plugin can optionally get global statistics (eg. the number of connections, JVM metrics, etc.) by parsing the metrics reported by AdminServlet of the Dropwizard application. Both, the access log and AdminServlet require a bit of configuration, see steps below.
+
+**The minimal supported version of Dropwizard is 1.0.0.**
+
+## Global statistics
+
+For gathering global statistics (eg. number of connections, etc.), the metrics page should be enabled.
+
+Add the following lines in your application YAML configuration file:
+
+{% highlight dropwizard %}
+server:
+  adminConnectors:
+    - type: http
+      port: 8081
+{% endhighlight %}
+
+Restart the application to apply these changes.  
+Provide the metrics URL, as you configured it on your server.
+
+## Log configuration
+
+Enable the Dropwizard aplication access log format. Add the following lines to your application YAML configuration file:
+
+{% highlight dropwizard %}
+server:
+  requestLog:
+    appenders:
+      - type: file
+        logFormat: '%h "%r" %D %s %b'
+        currentLogFilename: /var/log/dropwizard-app/access.log
+        archivedLogFilenamePattern: /var/log/dropwizard-app/accedd-%d.log.gz
+{% endhighlight %}
+
+Restart Dropwizard aplication to apply these changes.  
```

Comment:
'aplication' > 'application'

This is probably also wrong in the app, can you create a ticket for it?

Reply:
fixed